klog.go

```diff
@@ -1355,7 +1388,7 @@ func ErrorS(err error, msg string, keysAndValues ...interface{}) {
 	b.WriteString(fmt.Sprintf("err=%q", err.Error()))
 	kvListFormat(b, keysAndValues)
 
-	logging.println(errorLog, logging.logr, b)
+	logging.print(errorLog, logging.logr, b)
```

Comment:
Why use `logging.print` for ErrorS when InfoS uses `logging.println`?

Reply:
already fix it